mail/isync/Portfile

```diff
@@ -60,8 +59,11 @@ post-destroot {
     xinstall -m 644 -W ${worksrcpath} mbsync.plist ${destroot}${examples_dir}/${startupitem.plist}
 }
 
-configure.cppflags-append   -I${prefix}/include/db53
-configure.ldflags-append    -L${prefix}/lib/db53
+variant db5 description {Enable mdconvert compilation with Berkeley DB datatabase} {
```

Comment:
Please name the variant `bdb` (so that we can keep the name as is when we bump the dependency) and change the description to `Enable support for the alternative UID storage scheme` (see the `mbsync` `AltMap` store setting).

Reply:
updated